upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1222,6 +1222,120 @@ else
   echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="UPDATE_BSS_DATA_MASTERS"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
```

Comment:
It would be nice if we could invoke `csi` in a way to only generate the JSON files instead of redefining them here, but perhaps we can do that later via another JIRA?

While I don't expect us to update disk configurations all that much, but we will have to edit two places (csi and this) each time.

Reply:
CSI jira: https://jira-pro.it.hpe.com:8443/browse/MTL-2489

Reply:
Add a comment pointing to this JIRA. It'll ward off skeptics.